prac_06/guitars.py

```diff
@@ -0,0 +1,75 @@
+"""a program that keeps a list of user guitars and then displayes them once the user enters an empty string"""
+
+from guitar import Guitar
+
+
+def main():
+    """get guitars and add them to a list until nothing is entered then display all guitars """
+    guitars = []
+    print("My Guitars!")
+    name, year, cost = get_input()
+    guitar = Guitar(name, cost, year)
```

Comment:
This is less efficient than it should be. Why make a Guitar object when name is blank? Just follow the standard pattern - get name, then while it's not blank... problem solved.